src/App.tsx

```diff
@@ -1,26 +1,45 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { setTodos } from './features/todos';
+import { selectedTodo } from './features/currentTodo';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const dispatch = useDispatch();
+  const todoModal = useSelector(selectedTodo);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getTodos()
+      .then(todos => dispatch(setTodos(todos)))
+      .finally(() => setIsLoading(false));
+  }, [dispatch]);
```

Comment:
The `useEffect` hook has `dispatch` as a dependency. While this is technically correct, since `dispatch` is stable and doesn't change between renders, it might be unnecessary to include it in the dependency array. Consider removing it unless you have a specific reason to keep it, as it could lead to confusion.